repos/katello-repos.spec

```diff
@@ -1,6 +1,6 @@
 Name:           katello-repos
 Version:        2.4.0
-Release:        2.nightly%{?dist}
+Release:        1.rc1%{?dist}
```

Comment:
Following your previous comment, shouldn't this be 3.rc1 ? Wouldn't the same apply for going from 1.nightly to 2.rc1 ?  Not intending to nitpick but rather understand, get it right and ensure consistency.